content/docs/reference/routes/tls-upstream-allow-renegotiation.md

```diff
@@ -0,0 +1,21 @@
+---
+id: tls-upstream-allow-renegotiation
+title: TLS Upstream Allow Renegotiation
+keywords:
+- reference
+- TLS Upstream Allow Renegotiation
+---
+
+
+# TLS Upstream Allow Renegotiation
+- Config File Key: `tls_upstream_allow_renegotiation`
+- Type: `bool`
+- Default: `false`
+
+TLS Upstream Allow Renegotiation controls whether server-initiated TLS renegotiation is allowed for upstream servers.
```

Comment:
```suggestion
TLS Upstream Allow Renegotiation controls whether server-initiated TLS renegotiation is allowed for upstream servers. For more details, see Envoy's documentation on [`allow_renegotiation`](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/tls.proto).
```

Reply:
Might be worth pointing upstream (... or is it downstram? ) to Envoy's option her.e

Reply:
The text looks good, I'd only add a "description" to the front matter.